pkg/kclient/kclient.go

```diff
@@ -281,13 +279,15 @@ func (c *Client) IsSSASupported() bool {
 
 func (c *Client) checkIngressSupport() error {
 	var err error
-	c.isNetworkingV1IngressSupported, err = c.IsResourceSupported("networking.k8s.io", "v1", "ingresses")
-	if err != nil {
-		return fmt.Errorf("failed to check networking v1 ingress support %w", err)
-	}
-	c.isExtensionV1Beta1IngressSupported, err = c.IsResourceSupported("extensions", "v1beta1", "ingresses")
-	if err != nil {
-		return fmt.Errorf("failed to check extensions v1beta1 ingress support %w", err)
+	if c.checkIngressSupports {
```

Comment:
Once the check is done, I think we should mark it to false.

Reply:
Hmm i guess it makes sense, updating